pkg/hintrunner/zero/zerohint_others.go

```diff
@@ -144,3 +147,146 @@ func createMemEnterScopeHinter(resolver hintReferenceResolver, memset bool) (hin
 	}
 	return newMemEnterScopeHint(value, memset), nil
 }
+
+// SearchSortedLower hint searches for the first element in a sorted array
+// that is greater than or equal to a given key and returns its index
+//
+// `newSearchSortedLowerHint` takes 5 operanders as arguments
+//   - `arrayPtr` represents the offset in the execution segment of memory where starts the sorted array
```

Comment:
I'm also not sure about this information. I think it is an address to the base of the input array

Reply:
true, updated